app/Http/Controllers/WordsController.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\Categories;
+use App\Models\Choices;
+use App\Models\Words;
+use Illuminate\Http\Request;
+use App\Models\WordChoice;
```

Comment:
I just noticed now that your Models are mostly plural.

Please change all Models to singular form.

https://consoleinfotech.com/laravel-coding-convention.html